submissions/amooChristianah/database/TransportationCompany.sql

```diff
@@ -30,15 +32,51 @@ foreign key (passengers_id) references passengers(id)
 );
 
 select * from trip_details;
+select count(*) from trip_details;
 
 -- creating accident case table 
 
 CREATE TABLE accident_cases (
 id INT NOT NULL AUTO_INCREMENT,
 passenger_id INT NOT NULL,
-accident_status TINYINT(1) NOT NULL,
+accident_status TINYINT NOT NULL,
 PRIMARY KEY (id),
 foreign key (passenger_id) references passengers(id)
 );
 
-select * from accident_cases
+select * from accident_cases;
+select count(*) from accident_cases;
+
+
+-- QUERYING THE TRANSPORT DATABASE --
+#1. total number of survived passengers
+	SELECT COUNT(*) FROM accident_cases WHERE accident_status > 0;
+    -- Result = 342
+    
+#2. total number of passengers who did not survive
+	SELECT COUNT(*) FROM accident_cases WHERE accident_status = 0;
+    -- Result = 549
+    
+#3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherboug
+	SELECT passengers.full_name, passengers.sex FROM passengers 
+    LEFT JOIN trip_details ON passengers.id = trip_details.passengers_id
+    WHERE age < 27 and embarkation_point != 'S';
+    
+#4. Numbers of passengers that embarked at Southampson and survived
+	SELECT COUNT(*) FROM accident_cases
+    LEFT JOIN trip_details ON accident_cases.id = trip_details.passengers_id 
+    WHERE accident_status > 0 AND embarkation_point like 'S%';
+    -- Result = 218
+    
+#5. The id, name and the total number of passengers who paid a fare greater than $100 and above at the age od 35 with siblings and spouses on board
+	SELECT passengers.id, passengers.full_name,
+    COUNT(*) OVER () AS total_count  FROM passengers 
```

Comment:
There is no need to look for total_count here when you are already doing that in `5b`